.buildkite/longruns/pipeline.yml

```diff
@@ -63,8 +81,38 @@ steps:
         agents:
           slurm_ntasks: 32
 
+      - label: ":computer: no lim SSP baroclinic wave (ρe_tot) equilmoist high resolution"
+        command:
+          - "mpiexec julia --project=examples examples/hybrid/driver.jl --moist equil --precip_model 0M --z_elem 45 --dz_bottom 30 --h_elem 16 --kappa_4 1e16 --dt 400secs --t_end 100days --dt_save_to_disk 10days --job_id longrun_no_lim_ssp_bw_rhoe_equil_highres --ode_algo SSP333 --apply_limiter false"
+          - julia --color=yes --project=examples post_processing/remap/remap_pipeline.jl --data_dir longrun_no_lim_ssp_bw_rhoe_equil_highres --out_dir longrun_no_lim_ssp_bw_rhoe_equil_highres
+        artifact_paths: "longrun_no_lim_ssp_bw_rhoe_equil_highres/*"
+        env:
+          CLIMACORE_DISTRIBUTED: "MPI"
+        agents:
+          slurm_ntasks: 32
+
+      - label: ":computer: SSP baroclinic wave (ρe_tot) equilmoist high resolution"
+        command:
+          - "mpiexec julia --project=examples examples/hybrid/driver.jl --moist equil --precip_model 0M --z_elem 45 --dz_bottom 30 --h_elem 16 --kappa_4 1e16 --dt 400secs --t_end 100days --dt_save_to_disk 10days --job_id longrun_ssp_bw_rhoe_equil_highres --ode_algo SSP333"
+          - julia --color=yes --project=examples post_processing/remap/remap_pipeline.jl --data_dir longrun_ssp_bw_rhoe_equil_highres --out_dir longrun_ssp_bw_rhoe_equil_highres
+        artifact_paths: "longrun_ssp_bw_rhoe_equil_highres/*"
+        env:
+          CLIMACORE_DISTRIBUTED: "MPI"
+        agents:
+          slurm_ntasks: 32
+
+      - label: ":computer: baroclinic wave (ρe_tot) equilmoist high resolution topography (dcmip)"
```

Comment:
Same as above, this job has been moved to "experimental", so we don't need it here.